kmp-launchpad-google-utils/src/commonMain/kotlin/com/bottlerocketstudios/launchpad/google/utils/data/network/service/apiKey/ApiKeyService.kt

```diff
@@ -0,0 +1,5 @@
+package com.bottlerocketstudios.launchpad.google.utils.data.network.service.apiKey
+
+interface ApiKeyService {
```

Comment:
Would this be better named "...Provider" instead of "...Service".  Service usually refers to a thing that makes network calls.